RUFAS/rufas_time.py

```
@@ -59,6 +59,11 @@ def current_julian_day(self) -> int:
         """
         return int(self.current_date.strftime("%j"))
 
+    @property
+    def day_of_year(self) -> int:
+        """Alias for current_julian_day — day of the calendar year (1–365)."""
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)